m4_sessions/1803-inning/group_6.md

```diff
@@ -1,13 +1,21 @@
 Group 6
 
-Group Members:
+Group Members: Jorge PerezGa, Parker Lindley, Jeff Bender
 
-Name of Session:
+Name of Session: Value of Introverts
 
-Session Description (Why should students come to this session):
+Session Description (Why should students come to this session): To gain a deeper understanding of the importance of introverts
+in a extrovert valued work place.
 
-Session Outcomes (What will students learn/leave with?):
+Session Outcomes (What will students learn/leave with?): Some actionable advice to add value as an introvert.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+5 min introduction / nametags
```

Comment:
I like this outline.  This is an excellent talk: https://www.ted.com/talks/susan_cain_the_power_of_introverts

Also take a look at the introverts session from the last inning.  It was really well received by students because it was SUPER ACTIONABLE.  It wasn't just students "talking" at the group, but they tested things and tried them out.